Makefile

```
@@ -4,3 +4,8 @@ test:
 	@docker compose exec test-backend \
 		go test -v -p=1 ./test/e2e/e2e_test.go
 	@docker compose -f compose.test.yml down
+
+.PHONY: test-cover
+test-cover:
+	@cd backend; go test -v -p=1 -cover -tags="unit_test" ./... -coverprofile=cover.out
```

動かしてみたけど、こうする必要ありそう？？（自分の環境ではこうしないと出なかった）

```suggestion
	@cd backend; go test -v -p=1 -cover -tags="unit_test" ./... -coverprofile=cover.out; go tool cover -html=cover.out -o cover.html
```

---

直ってた！LGTM!